.travis.yml

```diff
@@ -1,20 +1,17 @@
 dist: trusty
-sudo: required
 language: rust
 
 os:
   - linux
 
 rust:
-  - stable
-  - beta
   - nightly
 
-matrix:
+jobs:
   include:
   - os: osx
     osx_image: xcode11.3
-    rust: stable
+    rust: nightly
```

Comment:
are we only able to compile with rust nightly now? If it is true, is it only because of va_list, or is there something else requires nightly build?

Reply:
https://github.com/tikv/rust-rocksdb/blob/master/librocksdb_sys/src/lib.rs#L14
This is not a stable feature